src/__swaps__/utils/numbers.ts

```diff
@@ -307,3 +342,23 @@ export const convertDecimalFormatToRawAmount = (value: string, decimals = 18): s
   new BigNumber(value).multipliedBy(new BigNumber(10).pow(decimals)).toFixed(0);
 
 export const fromWei = (number: BigNumberish): string => convertRawAmountToDecimalFormat(number, 18);
+
+const decimalSeparator = '.';
+const lessThanPrefix = '<';
```

Comment:
does lessThan changes between langs?

Reply:
dont think so, i just copied this util from another file. i can get rid of these constants